osquery/tables/applications/browser_utils.cpp

```diff
@@ -172,52 +172,104 @@ void genExtension(const std::string& uid,
 }
 
 QueryData genChromeBasedExtensions(QueryContext& context,
-                                   const fs::path& sub_dir) {
+                                   std::vector<fs::path> chromePaths) {
```

Comment:
Using const references should be preferred when possible: `const std::vector<fs::path> &chrome_paths`